File: packages/core/test/fixtures/regex/rust/launchdarkly/positive/basic.rs
use launchdarkly_server_sdk::Client;

pub fn checkout(client: &Client, context: &Context) -> &'static str {
    let enabled = client.bool_variation(context, "CHECKOUT_V2", false);
    if enabled {
        return "v2";
    }
    "v1"
}

File: packages/core/test/fixtures/regex/cpp/launchdarkly/positive/basic.cpp
#include <launchdarkly/server_side/client.hpp>

int Checkout(launchdarkly::Client& client, launchdarkly::Context& context) {
    bool enabled = client.BoolVariation(context, "CHECKOUT_V2", false);
    if (enabled) {
        return 2;
    }
    return 1;
}
